### PR TITLE
Need to update blog link

### PR DESCRIPTION
**Pull request by TomonoriAbe**
Current hyperlink to Office blog is https://blog.office.com. When user try to access the URL, they will be redirected to localized blog, such as https://www.microsoft.com/ja-jp/microsoft-365/blog/. However, there is no localized Office blog. So current link need to be modified to point US blog URL (https://www.microsoft.com/en-us/microsoft-365/blog/).

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/stay-on-top-of-updates.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ With Office 365, you receive new product updates and features as they become ava
...
-|**Blogs** and **Community** <br/> |Visit Office Blogs, Microsoft Community, and Microsoft Tech Community to learn more details about changes in Office 365 and share experiences with other users.  <br/> |Visit [Office Blogs](https://blogs.office.com/). Visit [Microsoft Community](https://answers.microsoft.com). Visit [Microsoft Tech Community](https://techcommunity.microsoft.com).  <br/> |
+|**Blogs** and **Community** <br/> |Visit Office Blogs, Microsoft Community, and Microsoft Tech Community to learn more details about changes in Office 365 and share experiences with other users.  <br/> |Visit [Office Blogs](https://www.microsoft.com/en-us/microsoft-365/blog/). Visit [Microsoft Community](https://answers.microsoft.com). Visit [Microsoft Tech Community](https://techcommunity.microsoft.com).  <br/> |
```
Comment (mypil) [inline Office365-Admin/manage/stay-on-top-of-updates.md]:
```suggestion
|**Blogs** and **Community** <br/> |Visit Office Blogs, Microsoft Community, and Microsoft Tech Community to learn more details about changes in Office 365 and share experiences with other users.  <br/> |Visit [Office Blogs](https://www.microsoft.com/microsoft-365/blog/). Visit [Microsoft Community](https://answers.microsoft.com). Visit [Microsoft Tech Community](https://techcommunity.microsoft.com).  <br/> |
```

Comment (mypil) [reply Office365-Admin/manage/stay-on-top-of-updates.md]: @TomonoriAbe - Please merge this commit to remove "en-us" from the URL.

Comment (TomonoriAbe) [reply Office365-Admin/manage/stay-on-top-of-updates.md]: @mypil Thank you for confirmation. However, if user open https://www.microsoft.com/microsoft-365/blog/ with Japanese language browser, user will be redirected to https://www.microsoft.com/ja-jp/microsoft-365/blog/, then open blank page. This is the same situation of original problem. That's why I request to modify to en-us link globally.

Comment (JohanFreelancer9) [reply Office365-Admin/manage/stay-on-top-of-updates.md]: I concur. Experienced the same result in Vietnam.

Comment (mypil) [reply Office365-Admin/manage/stay-on-top-of-updates.md]: @TomonoriAbe - Alright, we'll let the author check and decide on merging this PR. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. Please take note of the issue stated by @TomonoriAbe. 

Thank you.

cc: @kenwith

**Pull request merged**